module_4/config/status_codes.js

```diff
@@ -0,0 +1,10 @@
+export const statusCodes = {
```

Comment:
FYI:
https://nodejs.org/api/http.html#http_http_status_codes - native http nodejs' module contains such info)

Reply:
sorry, didn't get this
if i understand correctly we are adding status codes like in this answer from StackOverflow 
![image](https://user-images.githubusercontent.com/51543491/125793234-027e0bd4-2778-446d-b43a-4ba6d696af6c.png)
and purpose of object using is handling with project eslitlint rule "no magic numbers"

Reply:
sure, we add it, but in native module those constants are present

Reply:
you can use it or you can use your own

Reply:
found it, thanks